x/application/types/application.go

```diff
@@ -13,13 +13,24 @@ func (s *Application) IsUnbonding() bool {
 	return s.UnstakeSessionEndHeight != ApplicationNotUnstaking
 }
 
+// HasPendingTransfer returns true if the application has begun but not completed
```

Comment:
"Stake transfer" is not accurate. The entire application state is being transferred (incl. delegations). This is a method defined on the `Application` type, I don't see how this could be confused with other kinds of transfers.

Reply:
I discovered some files which I failed to rename in #788, I've completed the renaming here.